src/include/pbs_nodes.h

```diff
@@ -468,6 +468,7 @@ extern 	int	fix_indirectness(resource *, struct pbsnode *, int);
 extern	int	chk_vnode_pool(attribute *, void *, int);
 extern	void	free_pnode(struct pbsnode *);
 extern	int	save_nodes_db(int, void *);
+extern void propagate_socket_licensing(mominfo_t *, int);
```

Comment:
Here also align with the previous prototype declarations.

Reply:
Addressed with 522ed64459cc198b7348b455fd1e4882510691db